test/unit/test_grid_extrusion.py

```diff
@@ -604,56 +604,56 @@ def test_single_fracture(self):
         mg_new = d["mortar_grid"]
 
         # Check that the old and new grid have
-        bound_faces_old = mg.master_to_mortar_int().tocoo().col
+        bound_faces_old = mg.primary_to_mortar_int().tocoo().col
         # We know from the construction of the grid extruded to 3d that the faces
         # on the fracture boundary will be those in the 2d grid stacked on top of each
         # other
         known_bound_faces_new = np.hstack(
             (bound_faces_old, bound_faces_old + g_2.num_faces)
         )
-        bound_faces_new = mg_new.master_to_mortar_int().tocoo().col
+        bound_faces_new = mg_new.primary_to_mortar_int().tocoo().col
         self.assertTrue(
             np.allclose(np.sort(known_bound_faces_new), np.sort(bound_faces_new))
         )
 
-        low_cells_old = mg.slave_to_mortar_int().tocoo().col
+        low_cells_old = mg.secondary_to_mortar_int().tocoo().col
```

Comment:
secondary_faces_old

Reply:
This can also be tackled in a different PR. There will be just too many instances of low and high.

Reply:
TODO: See #478

Reply:
Unresolved, just to say that I agree with with postponing this.